docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,265 @@
+###############################################
+Compute an expectation value with ``Estimator``
+###############################################
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the :class:`~qiskit.primitives.Estimator` primitive.
+
+.. note::
+
+    While this guide uses Qiskit’s reference implementation, the ``Estimator`` primitive can be run with any provider using :class:`~qiskit.primitives.BackendEstimator` .
+    
+    .. code-block::
+
+        from qiskit.primitives import BackendEstimator
+        from qiskit_provider import QiskitProvider
```

Comment:
Maybe a way to signal that we are not referring to a literal module called like that?
```suggestion
        from <some_qiskit_provider> import QiskitProvider
```